index.js

```diff
@@ -101,9 +101,9 @@ function requestbody(opts) {
           ctx.req.body = body.fields;
           ctx.req.files = body.files;
         } else if (opts.includeUnparsed) {
-          ctx.req.body = body.parsed;
+          ctx.req.body = body.parsed || {};
           if (! ctx.is('text')) {
-            ctx.request.body[symbolUnparsed] = body.raw;  
+            ctx.req.body[symbolUnparsed] = body.raw;  
```

Comment:
This was wrong in the first place. We were supposed to be patching Node here not Koa.